sw/device/sca/lib/sca.h

```diff
@@ -175,6 +175,7 @@ uint32_t next_lfsr(void);
  * TODO: Remove once there is a CHECK_DIF_OK version that does not require test
  * library dependencies.
  */
+#if OT_IS_ENGLISH_BREAKFAST
```

Comment:
Can you update the comment to describe why this is ok now, given that `CHECK_DIF_OK()` still calls into `test_status_set()`?

Reply:
Thanks for flagging this. It turned out that the `#if` I added didn't work for `sha3_serial`. But now `aes_serial` uses `aes_testutils` which needs the test library dependency and thus the macro, I don't see a way around having the test library dependency. I've thus now removed the definition of the `CHECK_DIF_OK` macro here. Please let me know if you don't agree with this or see a better way.

Reply:
ack. SGTM